app/views/user/teams/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_tag(user_teams_path, method: :put) do %>
+  <div class="field-wrapper">
+    <%= label_tag :team_slug, 'Teams', class: 'col-sm-2 control-label' %>
+    <div class="col-sm-10">
+      <%= select_tag :team_slug, options_from_collection_for_select(@teams, 'slug', 'name'), disabled: @teams.blank? %>
+    </div>
+  </div>
+  <div class="btn-group actions pull-right">
+    <%= link_to t('teams.create'), new_team_path, class: "btn btn-default" %>
+    <%= submit_tag t('teams.select'), class: 'btn btn-primary', disabled: @teams.blank?  %>
```

Comment:
A partial shouldn't depend on a instance var (the `@teams`), pass it explicitly when  rendering this partial, like this:

```erb
   <%= render 'form', teams: @teams %>
```

This will make this partial more reusable.